next.config.js

```diff
@@ -33,6 +33,10 @@ const nextConfig = {
         protocol: "https",
         hostname: "getstream.io",
       },
+      {
+        protocol: "http",
```

Comment:
No actually. Cloudinary api uses http currently

Reply:
That is the api request url. I was talking about the url where the images are being saved. Currently those urls use http. It will throw error if used https.